geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -228,6 +231,31 @@ private void validateScoreIsDouble(byte[] score) {
     return members.get(member);
   }
 
+  byte[] zincrby(Region<RedisKey, RedisData> region, RedisKey key, byte[] increment,
+      byte[] member) {
+    byte[] byteScore = members.get(member);
+    double incr = processIncrement(Coder.bytesToString(increment).toLowerCase());
+
+    if (byteScore != null) {
+      double score = Coder.bytesToDouble(byteScore);
+      if (Double.isInfinite(score) && Double.isInfinite(incr)) {
+        throw new NumberFormatException(RedisConstants.ERROR_OPERATION_PRODUCED_NAN);
+      }
+      incr = score + incr;
+    }
```

Comment:
This would cause the addition of +inf + +inf (or -inf + -inf) to also result in the NAN error, but it shouldn't (should just be +inf or -inf respectively).  I'd prefer it if we checked the `incr` variable after doing the operation.  Double actually has an `isNan` method:
```
if(Double.isNaN(incr)) {
        throw new ArithmeticException(ERROR_OPERATION_PRODUCED_NAN);
      }
 ```